featurebyte/models/base.py

```diff
@@ -19,18 +19,18 @@
 
 Model = TypeVar("Model", bound="FeatureByteBaseModel")
 
-DEFAULT_CATALOG_ID = ObjectId("63eda344d0313fb925f7883a")
-ACTIVE_CATALOG_ID: ObjectId = DEFAULT_CATALOG_ID
+DEFAULT_CATALOG_ID = ObjectId("23eda344d0313fb925f7883a")
```

Comment:
Change the value of `DEFAULT_CATALOG_ID` so existing "default" catalog becomes a regular catalog.

The new `DEFAULT_CATALOG_ID` value is only used as a parameter to initialize catalog-specific services during data migration and is not used by an actual catalog.